tests/context/test_vtlookupv3.py

```diff
@@ -161,22 +160,6 @@ def close(self):
         return None
 
 
-def test_init_vt_lookup_class():
```

Comment:
Was this deleted on purpose?

Reply:
Yes it was. It was a trivial test and needed some nasty fix to instantiate VTLookup in an async method. 
I didn't think the effort to change the test was worth the value that the test brought.